Extensions/TweenBehavior/JsExtension.js

```diff
@@ -0,0 +1,524 @@
+/**
+ * This is a declaration of an extension for GDevelop 5.
+ *
+ * ℹ️ Run `node import-GDJS-Runtime.js` (in newIDE/app/scripts) if you make any change
+ * to this extension file or to any other *.js file that you reference inside.
+ *
+ * The file must be named "JsExtension.js", otherwise GDevelop won't load it.
+ * ⚠️ If you make a change and the extension is not loaded, open the developer console
+ * and search for any errors.
+ *
+ * More information on https://github.com/4ian/GDevelop/blob/master/newIDE/README-extensions.md
+ */
+module.exports = {
+  createExtension: function (_, gd) {
+    const extension = new gd.PlatformExtension();
+    extension
+      .setExtensionInformation(
+        "Tween",
+        _("Tween animation"),
+        _("Animate object properties via tweening."),
+        "Matthias Meike",
+        "Open source (MIT License)"
+      )
+      .setExtensionHelpPath("/all-features/tween");
+
+    var tweenBehavior = new gd.BehaviorJsImplementation();
+
+    tweenBehavior.updateProperty = function (
+      behaviorContent,
+      propertyName,
+      newValue
+    ) {
+      if (propertyName === "My first property") {
+        behaviorContent.property1 = newValue;
+        return true;
+      }
+      if (propertyName === "My other property") {
+        behaviorContent.property2 = newValue === "1";
+        return true;
+      }
+
+      return false;
+    };
+
+    tweenBehavior.getProperties = function (behaviorContent) {
+      var behaviorProperties = new gd.MapStringPropertyDescriptor();
+
+      behaviorProperties.set(
+        "My first property",
+        new gd.PropertyDescriptor(behaviorContent.property1)
+      );
+      behaviorProperties.set(
+        "My other property",
+        new gd.PropertyDescriptor(
+          behaviorContent.property2 ? "true" : "false"
+        ).setType("Boolean")
+      );
+
+      return behaviorProperties;
+    };
+
+    tweenBehavior.setRawJSONContent(
+      JSON.stringify({
+        property1: "Initial value 1",
+        property2: true
+      })
+    );
+
+    const behavior = extension
+      .addBehavior(
+        "TweenBehavior",
+        _("Tween animation"),
+        "TweenBehavior",
+        _("Tween animation"),
```

Comment:
"Tween anination" is displayed on two levels in the list of conditions/actions before the defined actions are shown:
-  Tween animation
  - Tween animation
    - Add XXX tween
    - Add YYY tween
    - etc.

How can I get rid of this?

Reply:
Using an empty group: "" should work

Reply:
And "Tween animation/Variables" should be simply "Variables", etc.